lib/screens/explore_screens/vote_screens/vote/vote_screen.dart

```diff
@@ -23,12 +24,14 @@ class VoteScreen extends StatelessWidget {
         child: Scaffold(
           appBar: AppBar(
             actions: [
-              IconButton(
-                onPressed: () {
-                  NavigationService.of(context).navigateTo(Routes.delegate);
-                },
-                icon: SvgPicture.asset('assets/images/explore/delegate.svg'),
-              ),
+              if (remoteConfigurations.featureFlagDelegate)
```

Comment:
Do not sue re repos directly in the UI

The page state must have this information. 

Look at how the recoveryPhraseFeatureFlag is used

Reply:
done